src/auth/index.js

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+// export const BASE = 'https://warm-savannah-72362.herokuapp.com/api'
+export const BASE = 'http://localhost:5000/api'
+
+export const callApi = async ({method, body, url, token}) => {
```

Comment:
Either use the function, or remove it!  I would love to see this get used... it's a good function...